docs/source/quickstart.rst

```diff
@@ -151,9 +151,17 @@ Conda installs the catalog file in this package to ``$CONDA_PREFIX/share/intake/
 The global catalog is a union of all catalogs installed in the conda/virtualenv environment and also any catalogs
 installed in user-specific location.
 
+
+Adding Data Source Packages using the Intake path
+-------------------------------------------------
+Intake checks the environment variable `INTAKE_PATH` for a colon separated list of paths (semicolon on windows) to search for catalog files.
```

Comment:
Can you please update this to specify the config file option, and that the paths should be *local* (for now). Actually, should the config variant strictly follow the same syntax, or could it have normal YAML syntax, in which case it could easily be remote paths too?
In this one specific case, do you think that the environment variable should override, or is there an argument that it should only add? Normal BASH path lists do override, even if the typical use case includes the list as defined so far.

Reply:
I was thinking the same thing. The config option could be a YAML list, and when the path env var gets unpacked we then turn it into a list. I do think that the env var should override the config.